src/js/lib/admin-page/style.scss

```diff
@@ -0,0 +1,166 @@
+$gray: #777;
+
+#wp-parsely_version {
+	color: $gray;
+	font-size: 12px;
+	margin-left: 1em;
+}
+
+.help-text {
+	width: 75%;
+}
+
+.controls {
+	display: flex;
+	flex-direction: row;
+}
+
+.nav-control {
+	cursor: pointer;
+	justify-content: space-between;
+	height: 2vh;
+	font-size: larger;
+	padding: 1em;
+}
+
+.inactive {
+	display: none;
+}
+
+span.active {
+	font-weight: bolder;
+	text-decoration: underline;
+}
+
+.setting-item {
+	display: inline-flex;
+	flex-direction: row;
+	justify-content: space-between;
+	width: 50%;
+}
+
+.setting-item--control {
+	width: 100%;
+	max-width: 300px;
+}
+
+.site-detail {
+	display: flex;
+	flex-direction: column;
+	width: 100%;
+	max-width: 300px;
+	padding-bottom: 1em;
+}
+
+.details-info {
+	display: flex;
+	flex-direction: column;
+	border: 1px solid #ccc;
+	width: 100%;
+	padding: 5% 20% 5% 5%;
+}
+
+.details-label {
+	width: 50%;
+	font-size: large;
+	padding-bottom: 1em;
+}
+
+.code {
+	font-family: source-code-pro, Menlo, Monaco, Consolas, 'Courier New', monospace;
+}
+
+.modal {
+	width: 500px;
+	left: 50%;
+	top: 50%;
+	position: absolute;
+	background: white;
+	border: 1px solid #ccc;
+	transition: 1.1s ease-out;
+	transform: translate(-50%, -50%);
+	box-shadow: -2rem 2rem 2rem rgb(0 0 0 / 20%);
+	padding: 2em;
+	filter: blur(0);
+	opacity: 1;
+	visibility: visible;
+	z-index: 9999;
+}
+
+.wipe-metadata-button {
+	margin-top: 1em;
+	margin-bottom: 1em;
+	width: 100%;
+	max-width: 300px;
+	height: 30px;
+	color: #b0443f;
+}
+
+.wipe-metadata-button:hover {
+	color: #ffffff;
+	background-color: #b0443f;
+}
+
+.wipe-metadata-label {
+	padding-top: 1em;
+}
+
+.container {
+	width: 75%;
+	display: flex;
+	flex-direction: row;
+}
+
+.site-details-container {
+	margin-top: 1em;
+	display: flex;
+	flex-direction: column;
+}
+
+.wipe-metadata-container {
+	display: flex;
+	flex-direction: column;
+	margin-bottom: 1em;
+}
+
+.wipe-metadata-controls {
+	display: flex;
+	flex-direction: column;
+}
+
+.modal-container {
+	width: 100%;
+	height: 100%;
+	position: absolute;
+}
+
+.modal-background {
+	background-color: rgba(black, 0.4);
+	width: 100%;
+	height: 100%;
+	position: absolute;
+	margin-right: 2em;
+	animation: metadata-fade-in 150ms linear;
+}
+
+.copy-cta {
+	cursor: pointer;
+}
+
+#wp-parsely-react-entrypoint {
```

Comment:
We should probably wrap most of the above in this qualifier to be sure we're only applying styles to our own elements.

Reply:
Moved!

Reply:
Moved everything into the qualifier! it's all under the `#wp-parsely-react-entry-point` declaration, and everything still looks styled!